src/index.js

```diff
@@ -216,9 +216,10 @@ walletRouter.get('/address',
     return res.status(400).json(validationResult);
   }
   const wallet = req.wallet;
-  const index = req.query.index || null;
```

Comment:
Nice catch! Maybe add a test to this in the other issue, right?

Reply:
It's important. I will reference this comment here in the test PR